app/commands/helpers/incident_helper.py

```diff
@@ -116,7 +116,11 @@ def archive_channel_action(client, body, ack):
         )
         log_to_sentinel("incident_channel_archive_delayed", body)
     elif action == "archive":
-        client.conversations_archive(channel=channel_id)
+        # get the current chanel id and name and make up the body with those 2 values
+        body = {"channel_id": channel_id, "channel_name": body["channel"]["name"]}
```

Comment:
Small favour - could you use a different variable here so that it doesn't override the request's original `body` values.  That way we'll still have the username, etc. when we log to Sentinel below on line 124.

Reply:
Good point @patheard! This has been changed.